src/main/java/org/broadinstitute/hellbender/utils/locusiterator/LocusIteratorByState.java

```diff
@@ -366,40 +345,4 @@ private boolean dontIncludeReadInPileup(final GATKRead rec, final long pos) {
         return ReadUtils.isBaseInsideAdaptor(rec, pos);
     }
 
-    /**
```

Comment:
Not in this PR, but `dontIncludeReadInPileup` can be made static. IntelliJ also says that we can get rid of the `samples` field; just pass a new copy to the `ReadStateManager` in line 205.

Reply:
done and done